### ISSUE TITLE
dotnet-runtimeinfo exception on wsl1

### ISSUE THREAD

**Issue by pengweiqhca** [dotnet/core#5584]
``` bash
**.NET information
Version: 3.1.10
FrameworkDescription: .NET Core 3.1.10
Libraries version: 3.1.10-servicing.20516.1
Libraries hash: d989459717cbce34262060c4b8b949185033e379

**Environment information
OSDescription: Linux 4.4.0-19041-Microsoft #488-Microsoft Mon Sep 01 13:43:00 PST 2020
OSVersion: Unix 4.4.0.19041
OSArchitecture: X64
ProcessorCount: 12

**CGroup info**
Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path '/sys/fs/cgroup/cpu/cpu.cfs_quota_us'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
   at System.IO.FileStream.OpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
   at System.IO.StreamReader..ctor(String path, Encoding encoding)
   at System.IO.File.InternalReadAllLines(String path, Encoding encoding)
   at System.IO.File.ReadAllLines(String path)
   at RuntimeEnvironment.Program.Main(String[] args) in /_/samples/dotnet-runtimeinfo/Program.cs:line 29
Aborted (core dumped)
```

**Comment by mairaw** [comment]
Is this something you can help with @richlander?

**Comment by richlander** [comment]
Interesting. I can update the tool to make it more resilient to this case. Thanks for the report!

**Comment by richlander** [comment]
Try version [1.0.3](https://www.nuget.org/packages/dotnet-runtimeinfo/1.0.3) of the tool. I tested in on WSL1 and it works now.

Fixed in #5592.
